v3/src/global.d.ts

```diff
@@ -4,6 +4,14 @@ declare module "*.png";
 declare module "*.svg";
 declare module "d3-v6-tip";
 
+declare module "color-parse" {
+  export default function parse(cstr: string): {
```

Comment:
Any way to use the `ParseColorResult` interface defined in `color-parse-strict.ts` for the return type here?